pennylane_lightning/core/lightning_base.py

```diff
@@ -57,6 +58,10 @@ class LightningBase(Device):
     """
 
     # pylint: disable=too-many-instance-attributes
+    pennylane_requires = ">=0.40"
+    version = __version__
+    author = "Xanadu Inc."
+    short_name = "lightning.base"
```

Comment:
What are these attributes? I don't think it's a good idea a short name and author to this base class as it's not supposed to be used as a standalone PL's plugin.
```suggestion
```